packages/riverpod_lint_flutter_test/test/assists/convert_to_widget/convert_to_consumer_widget.json

```diff
@@ -99,18 +99,23 @@
           "fileStamp": 0,
           "edits": [
             {
-              "offset": 993,
-              "length": 174,
+              "offset": 1167,
+              "length": 0,
               "replacement": ""
```

Comment:
Sounds incorrect. We're now replacing nothing with nothing

Reply:
Fixed in d5b8b6da2bf5c170af934f9b62a26d4c4aa751c5 .